designs/administrator/future-openshift/user-preferences/index.md

```diff
@@ -0,0 +1,62 @@
+---
+parent: Administrator
+version: 4.9
+---
+
+# Adding User Preferences to OpenShift Console
+
+We are finally adding user preferences to the console in 4.9. This enhancement will allow individual users to set preferences across the console, providing a customizable experience.
+
+In 4.9 user preferences will consist of two tabs:
+1. General preferences
+2. Language and region preferences
+
+All changes in user preferences will be autosaved.
```

Comment:
I was wondering if there was any precedent in the console for autosaving today? Should maybe the fact that these will be autosaved be called out in the UI text at the top of the page? Or some "Saving..." text toast after each dropdown selection?

Reply:
We talked about a toast, but for the MVP decided not to go there since we don't totally have the guidelines established. But I think we could add a sentence to the top of the page if you think that would be helpful - like your changes will be autosaved or something along those lines?

Reply:
Only since there doesn't seem to be any existing precedent for users to expect their changes to be autosaved when moving through the console, I do think it would be worthwhile to include "Any changes will be autosaved." or something.